ui/src/components/PlayerColumn.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { FC } from 'react';
+import ReactPlayer from "react-player";
+import { Radio } from '../lib';
+import { Navigation } from './Navigation';
+import { HelpMenu } from './HelpMenu';
+import { useAppSelector, useAppDispatch } from '../app/hooks';
+import {
+  selectSpinUrl,
+  selectSpinTime,
+  selectTunePatP,
+  selectViewers
+} from '../features/station/stationSlice';
+import {
+  setPlayerReady,
+  setPlayerInSync,
+  setHelpMenuOpen,
+  setHelpMenuTop,
+  setHelpMenuLeft,
+  selectPlayerInSync,
+  selectPlayerReady,
+  selectHelpMenuOpen,
+  selectHelpMenuTop,
+  selectHelpMenuLeft
+} from '../features/ui/uiSlice';
+
+interface IPlayerColumn {
+  our: string;
+  radio: Radio;
+  tuneTo: ((patp:string | null) => void);
+}
+
+export const PlayerColumn: FC<IPlayerColumn> = (props: IPlayerColumn) => {
+
+  const {our, radio, tuneTo} = props;
+
+  const spinUrl = useAppSelector(selectSpinUrl);
+  const spinTime = useAppSelector(selectSpinTime);
+  const tunePatP = useAppSelector(selectTunePatP);
+  const playerReady = useAppSelector(selectPlayerReady);
+  const playerInSync = useAppSelector(selectPlayerInSync);
+  const viewers = useAppSelector(selectViewers);
+  const helpMenuOpen = useAppSelector(selectHelpMenuOpen);
+  const helpMenuTop = useAppSelector(selectHelpMenuTop);
```

Comment:
move some of this out of global state. eg helpMenuTop is only needed locally in this component